### PR TITLE
Add organic inspection schemas

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR adds a schema for organic inspections as well as a supporting schema.

Not covered in this PR but in consideration for later development:
* An organic certification credential
* Sample analysis information
* Onsite interview information
* An organic inspection report schema & credential (within which organic inspection would be included)

### CODE REVIEW

File: docs/openapi/components/schemas/common/OrganicInspection.yml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+  samplingDetails:
+    title: Sampling Details
+    description: If pesticide residue sampling was conducted, please provide details.
+    type: string
+    $linkedData:
+      term: samplingDetails
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#content
+  introductionOperationDescription:
+    title: Introduction / Operation Description
+    description: Describe conditions at the time of inspection, and any general summary comments about the operation needed to complement the description in the client's OSP. Note to which standards (NOP) compliance is verified, including any additional certifications (international standards, GAPs, etc).
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/OrganicInspection.yml:107]:
```suggestion
    description: Describe conditions at the time of inspection, and any general summary comments about the operation needed to complement the description in the client's OSP. Note to which standards (NOP) compliance is verified, including any additional certifications (international standards, GAPs, etc.).
```

File: docs/openapi/components/schemas/common/OrganicInspection.yml [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+    type: string
+    $linkedData:
+      term: introductionOperationDescription
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#description
+  resolutionIssuesActionItems:
+    title: Resolution of Issues and Action Items
+    description: List any previously identified issue, non-compliance, corrective action required, or request for further information provided by the Certifier, a description of how it has been addressed; whether is it resolved, or is pending further action.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/OrganicInspection.yml:114]:
```suggestion
    description: List any previously identified issue, non-compliance, corrective action required, or request for further information provided by the Certifier, a description of how it has been addressed, i.e., whether is it resolved, or is pending further action.
```

File: docs/openapi/components/schemas/common/OrganicOSPSectionReview.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  OSPSectionCode:
+    title: OSP Section Code
+    description: A code corresponding to the relevant OSP section.
+    type: string
+    $linkedData:
+      term: OSPSectionCode
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#standard
+  resultCode:
+    title: Result Code
+    description: Must be one of "C" (Compliant), "NC" (Not Compliant), "MIN" (More Information Needed), or "R (Reminder), or "NA" (Not Applicable).
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/OrganicOSPSectionReview.yml:27]:
```suggestion
    description: Must be one of "C" (Compliant), "NC" (Not Compliant), "MIN" (More Information Needed), "R" (Reminder), or "NA" (Not Applicable).
```

File: docs/openapi/components/schemas/common/OrganicInspection.yml (hunk 2 above)
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/OrganicInspection.yml:114]: "whether it is resolved, or is pending further action" is not a summary of the previous statement, it's an additional thing to note in that field.

Whether a semicolon is technically correct for this is debatable, but a) it gets the point across and is appropriate for terse delivery, and b) I'm copying all of this directly from the source material.

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/common/OrganicInspection.yml (hunk 2 above)
Comment (TallTed) [reply docs/openapi/components/schemas/common/OrganicInspection.yml:114]: Regrettably, plenty of the source material includes language issues/errors.  Where we can do better, I think we should.

Now I *think* I understand what they meant to say, here's a revised suggestion --
```suggestion
    description: List any previously identified issue, non-compliance, corrective action required, or request for further information provided by the Certifier; a description of how each has been addressed; and whether each has been resolved or is pending further action.
```

**Comment by nissimsan** [comment]
@rhofvendahl , please work in @TallTed's suggestions and merge when done.

**Pull request merged**